Horizon/project.yml

```diff
@@ -6,6 +6,9 @@ options:
   deploymentTarget:
     iOS: 17.0
   createIntermediateGroups: true
+configFiles:
+  Debug: Horizon/Config/Debug.xcconfig
+  Release: Horizon/Config/Release.xcconfig
```

Comment:
We want to add these to `project-ci.yml` because CI calls the `make sync-ci` command.